frontend/src/static/js/pages/styles/ErrorPage.scss

```
@@ -80,10 +80,10 @@
 			&.error-btn-primary {
 				background: var(--btn-primary-bg-color);
 				color: var(--btn-color);
-				border-color: var(--btn-border-color);
+				border-color: transparent;
 
 				> .btn-icon {
-					color: var(--btn-primary-icon-color);
+					color: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)